operator/roles/default/kiali-deploy/templates/openshift/deployment.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
   labels:
     app: kiali
     version: {{ kiali_vars.deployment.version_label }}
+  annotations:
```

Comment:
are you sure you want these here on the deployment itself? or on the pod (see template section below)